src/index.js

```diff
@@ -12,59 +32,93 @@ const handleErrorWithSource = async (source, fn) => {
 export default inputs => {
   const { query, params, body, headers, config } = inputs;
 
-  if (config !== undefined && Object.prototype.toString.call(config) !== "[object Object]") {
+  if (config !== undefined && Object.prototype.toString.call(config) !== '[object Object]') {
     throw { message: `Route config, expecting config to be an Object` };
   }
 
   const _config = {
     denyUnknown: [],
     httpErrorCode: 400,
     nextOnError: false,
+    alternate: [],
     ...config
   };
 
   return async (ctx, next) => {
+    let alternateSchema = {};
+    let alternateData = [];
```

Comment:
@lloria91 this variable is initiated as an array, but it seems that its value is an object?

Reply:
yes. after declaration it was changed to an object, so missed it ;)